src/Resources/contao/dca/tl_image_size.php

```diff
@@ -3,17 +3,15 @@
 namespace Merconis\Core;
 
 use Contao\Backend;
+use Contao\CoreBundle\DataContainer\PaletteManipulator;
 use Contao\Database;
 use Contao\DataContainer;
 use Contao\System;
 
-// Anpassung der Palette
-$GLOBALS['TL_DCA']['tl_image_size']['palettes']['default'] = str_replace
-(
-    'name,',
-    'name,merconis_alias,',
-    $GLOBALS['TL_DCA']['tl_image_size']['palettes']['default']
-);
+PaletteManipulator::create()
+    ->addField('merconis_alias', 'name')
```

Comment:
Der zweite Parameter von "addField" heißt "$parent" und z. B. in src/Resources/contao/dca/tl_content.php (selber PR) wird als zweiter Parameter auch die legend angegeben. Hier allerdings wird als zweiter Parameter "name" angegeben, was scheinbar das Feld ist, hinter dem im Code vor der Änderung das Feld "merconis_alias" platziert wurde.
Kann sein, dass das dennoch so richtig ist. Ich kennen den PaletteManipulator nicht gut. Sieht halt verdächtig aus. Bitte prüfen, ob hier ein Fehler vorliegt.

Reply:
Bei der DCA von tl_content legen wir eine neue legend an, dann muss man die gewünschten Felder der neuen legend anhängen.
Hier fügt man ein neues field einer bestehenden legend an und muss daher ein bestehenden field-Name angeben wo man es einhängen möchte.

Reply:
Okay, ich hatte mich da vom Parameter-Namen "$parent" wohl etwas ins Bockshorn jagen lassen.